src/api/mockpsp_api/v1/_swagger.json.tpl

```diff
@@ -0,0 +1,142 @@
+{
+  "openapi": "3.0.1",
+  "info": {
+    "title": "Pagopa PM Mock",
+    "version": "1.0.0"
+  },
+  "tags": [
```

Comment:
I don't know if we need to add `host` property like https://github.com/pagopa/pagopa-infra/blob/main/src/api/mockec_api/v1/_swagger.json.tpl#L9
Maybe we can try to apply and see if it works without this property setted